va_explorer/va_data_management/utils/coding.py

```diff
@@ -12,13 +12,51 @@
 from va_explorer.va_data_management.models import CauseOfDeath
 from va_explorer.va_data_management.models import VerbalAutopsy
 
+# NOTE: By default, VA Explorer runs InterVA5 (settings found in .env file)
+# To change coding algorithm, will need to update settings below and point
+# to that algorithm's service
 
 PYCROSS_HOST = os.environ.get('PYCROSS_HOST', 'http://127.0.0.1:5001')
 INTERVA_HOST = os.environ.get('INTERVA_HOST', 'http://127.0.0.1:5002')
 
-# TODO: settings need to be configurable
-ALGORITHM_SETTINGS = {'HIV': 'l', 'Malaria': 'l', 'groupcode': 'True'}
-
+# Param Setting value sets (used for validation)
+# TODO: add other algorithms' settings as we add support for them
+ALGORITHM_PARAM_OPTIONS = {
+    'INTERVA': {'HIV': ['h', 'l', 'v'],
+                'Malaria': ['h', 'l', 'v'],  
+                'groupcode': ['True', 'False']
+                }
+}
+
+# InterVA5 
+ALGORITHM_SETTINGS = {
+    # describes prevalence of HIV. Should be "h"(high),"l"(low), or "v"(very low).
+    'HIV': os.environ.get('INTERVA_HIV', 'h'),
+    # describes prevalence of Malaria. Should be "h"(high),"l"(low), or "v"(very low).
+    'Malaria': os.environ.get('INTERVA_MALARIA', 'l'),
+    # Whether to include group code in the output causes
+    'groupcode': os.environ.get('INTERVA_GROUPCODE', 'True')
+}
+
+# validates provided interva settings against interva5 param value sets
+def validate_algorithm_settings(settings=ALGORITHM_SETTINGS):
```

Comment:
Is settings being passed in for future work? Otherwise, `ALGORITHM_SETTINGS` is in the global context and can be used in the method as is.

Reply:
Not sure. Wanted to make the function flexible but don't think it people would use would use it in a one-off fashion. Will change to just use ALGORITHM_SETTINGS